flixel/util/FlxTimer.hx

```diff
@@ -232,11 +241,31 @@ class FlxTimerManager extends FlxBasic
 	 */
 	override public function update(elapsed:Float):Void
 	{
+		var timerLoops:Int;
```

Comment:
I prefer to declare local vars as close to where they're actually needed as possible.